### DISCUSSION TITLE
Is it supported by systemV without using some kind of wrapper or something?

### DISCUSSION THREAD

**Discussion by Raj2032** [AdnanHodzic/auto-cpufreq#450] [Q&A]
I want to install [Peppermint Devaun edition](https://peppermintos.com/guide/downloading/) which uses systemV rather than systemD, so I wanted to enquire if this is supported by systemV as well without using some kind of wrapper, as I know that with Gnome applications you have to use some kind of wrapper to get systemV working with Gnome applications so that is why I ask?

**Comment by AdnanHodzic** [comment]
There's already request for [Devuan support in issues](https://github.com/AdnanHodzic/auto-cpufreq/issues/29).

Since I don't use it and am very limited with available time to do this myself, if you or anyone else wants to give it a try in implementing this functionality, please give it a try and [contribute to the project](https://github.com/AdnanHodzic/auto-cpufreq/#code-contribution) and you will be credited for your work as part of [future release](https://github.com/AdnanHodzic/auto-cpufreq/releases).
